Shared/Samples/Find closest facility to multiple points/FindClosestFacilityToMultiplePointsView.swift

```diff
@@ -63,6 +63,14 @@ private extension FindClosestFacilityToMultiplePointsView {
         let map = {
             let map = Map(basemapStyle: .arcGISStreets)
             map.initialViewpoint = Viewpoint(latitude: 32.727, longitude: -117.175, scale: 144_400)
+            let envelope = Envelope(
+                xMin: -13049024,
+                yMin: 3856693,
+                xMax: -13039785,
+                yMax: 3862925,
+                spatialReference: .webMercator
+            )
+            map.maxExtent = envelope
```

Comment:
```suggestion
            map.maxExtent = .sanDiegoCityLimits
```

This envelope can be an extension to simplify the code. Also the name of the extension makes the purpose clear. X and y ranges should be used as well.

```swift
private extension Envelope {
    /// An envelope of the San Diego city limits.
    static var sanDiegoCityLimits: Envelope {
        Envelope(
            xRange: -13049024 ... -13039785,
            yRange: 3856693...3862925,
            spatialReference: .webMercator
        )
    }
}
```